src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -414,6 +418,12 @@ func NewStartCommand(
 		false,
 		"Enable overprovisioning",
 	)
+	command.Flags().BoolVar(
+		&enableWasm,
+		enableWasmFlag,
+		false,
+		"Enable coprocessors implementation",
```

Comment:
I think this should be the same as https://github.com/vectorizedio/redpanda/pull/1393/commits/c32bda131605c00defe79e70185faecd3ffe71f2#diff-24a1b5b910769f8aaa14ada82b9d18cec7b5d1618db88b23dc2c98275d582283R89